magmi/inc/magmi_mixin.php

```diff
@@ -25,7 +25,7 @@ public function __call($data, $arg)
         if (substr($data, 0, 8) == "_caller_") {
             $data = substr($data, 8);
         }
-        $ccallers = count($this->_callers);
+        $ccallers = empty($this->_callers) ? 0 : count($this->_callers);
```

Comment:
shoudlnt we just initialize the variable like `protected $_callers = array();`
Can you please check if this approach also work for you? sounds more elegant to me.

Reply:
see similar patch: https://github.com/dweeves/magmi-git/pull/587/files